plugins/content_distribution/providers/tvinci/TvinciDistributionPlugin.php

```diff
@@ -2,7 +2,8 @@
 /**
  * @package plugins.tvinciDistribution
  */
-class TvinciDistributionPlugin extends KalturaPlugin implements IKalturaPermissions, IKalturaEnumerator, IKalturaPending, IKalturaObjectLoader, IKalturaContentDistributionProvider, IKalturaConfigurator
+class TvinciDistributionPlugin extends KalturaParentContributedPlugin implements IKalturaPermissions, IKalturaEnumerator, IKalturaPending, IKalturaObjectLoader,
```

Comment:
You're right, and in fact it doesn't make sense to implement it as parent class.
If it's a standalone feature, implement it as a standalone plugin, or as a core feature, this way different plugins will be able to use it even if they already extend a different parent.